proposals/p0044.md

```diff
@@ -588,12 +585,7 @@ Disadvantages:
 -   Email notifications include only the lines of code affected, not what is
     being replied to. This will generally make it infeasible to get context from
     emails.
--   Comment threads sometimes make it unclear what's being replied to. for
-    example,
-    https://github.com/carbon-language/carbon-proposals/pull/5#discussion_r423401993
-    and
-    ttps://github.com/carbon-language/carbon-proposals/pull/5/files/a51ff951561accfb4aee403d7add6e8e69009ce1#r423401993
-    are equivalent, but the replied-to-comment is only visible in the file view.
+-   Comment threads sometimes make it unclear what's being replied to.
```

Comment:
I'd be fine to fix the typo of a missing `h` in the second of these, but beyond that I'd annotate and preserve the broken links here rather than deleting all of this text.

Reply:
Okay. I've restored and annotated the broken links that had no replacement.